tests/backup/backup_helper.go

```diff
@@ -2027,6 +2032,11 @@ func UpgradePxBackup(versionToUpgrade string) error {
 		return err
 	}
 
+	// Collect mongoDB logs once the postInstallHook is completed
```

Comment:
On second thoughts, I think this should be inside `ValidateAllPodsInPxBackupNamespace` because that is where we are checking if all the pods in px-backup namespace are up and running

Reply:
Done